### PR TITLE
Edit helpcommand

### LINKED ISSUE #81: As a user, I want to have a Help page that lists all commands so that I know how to navigate through the app

### PR DESCRIPTION
**Pull request by pennhanlee**
Closes #81 

12/10/20
- Created a more meaningful help page with Images as Example 
- Editted Copy URL to copy bookmark website address. 

13/10/20
- Resolve alignment problem with scrollpane having unnecessary border
- Resolve scroll position and set to start on top
- Resolve alignment issue with Text and Image for the examples

14/10/20
- Added separation between Headerbar and Content Body
- Updated UG and DG
- Updated new UI screenshots

*Starting the Help Page looks like this*
<p align="center">

</p>

*Scrolling the Help Page will leave the Header alone*
<p align="center">

</p>

### CODE REVIEW

**Comment by angrybunny123** [comment]
Is it possible to separate the header & the main body with a separation line? Other than that, LGTM!

**Review by angrybunny123 — COMMENTED** [review]
Is it possible to separate the header & the main body with a separation line? Other than that, LGTM!

File: src/main/java/seedu/bookmark/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,17 +18,66 @@
...
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2021s1-cs2103t-f13-2.github.io/tp/";
+    public static final String HEADER_MESSAGE = "Welcome to bookmark!\n"
+                                                + "Click on the Copy URL button for the link to our Website!\n";
+
+    public static final String ADD_INTRO = "You can now ADD a book! \n";
+    public static final String ADD_COMMAND = "add n/{Book Name} g/{Genre} t/{Tags}* t/{Tags}* "
```
Comment (pangpuncake) [inline src/main/java/seedu/bookmark/ui/HelpWindow.java]: Perhaps using the format provided in the [User Guide](https://github.com/AY2021S1-CS2103T-F13-2/tp/blob/master/docs/UserGuide.md) e.g. `add n/BOOK_TITLE tp/TOTAL_PAGES b/PAGE_NUMBER` would be more consistent?

**Review by pangpuncake — COMMENTED** [review]
Adding a separator between header and content like Kenneth said would be great!
Other than the formatting of the *_COMMAND, LGTM!

**Review by TanLeYang — COMMENTED** [review]
Could you change the screenshots to use the new UI? Also would be great if you could add a small section for the help command in the UG. Other than that LGTM!

**Comment by angrybunny123** [comment]
LGTM!

**Review by TanLeYang — APPROVED** [review]
LGTM!

**Pull request merged**